src/domain/insert_email.py

```diff
@@ -139,15 +144,16 @@ def send_email(self):
 
         credentials = keyring.get_credential("WADAS_email", "")
         sender = credentials.username
-        recipients = self.ui.textEdit_recipient_email.toPlainText().strip()
-        recipients_to_list = recipients.split(", ")
+        recipients_to_list = list() 
+        for recipient in self.ui.textEdit_recipient_email.toPlainText().strip().split(", "):
+            recipients_to_list.append(recipient)
```

Comment:
recipients = [recipient for recipient in self.ui.textEdit_recipient_email.toPlainText().strip().split(", ")]